docs/introduction/framework-extensibility.md

```diff
@@ -22,6 +22,8 @@ as well as a list of customizations that are not (and will not be) possible at a
 * open-box modifications in `project-base`
     * e.g. adding new entities, changing the FE design, customization of FE javascripts, adding new FE pages (routes and controllers), ...
 * [Hiding the existing features and functionality](https://github.com/shopsys/demoshop/pull/13)
+* add javascript into admin
+    * add it into directory `src/Shopsys/ShopBundle/Resources/scripts/custom_admin` and it will be loaded automatically
```

Comment:
add the new javascript files into the directory `src/Shopsys/ShopBundle/Resources/scripts/custom_admin` and they will be